### PR TITLE
Implements calculation of returning of investments

### PR DESCRIPTION
**Pull request by dmytrostriletskyi**

### CODE REVIEW

File: calculator/domain/roi.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+
+class Roi:
+    """
+    Implements returning on investment.
+    """
+
+    def __init__(self, economy: Economy, block_reward: BlockReward, block_producer: BlockProducer):
+        """
+        Calculator.
```
Comment (anastasiia-bilova) [inline calculator/domain/roi.py]:
```suggestion
        Constructor.
```

File: calculator/domain/roi.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+            active_block_producer_reward = ActiveBlockProducerReward(
+                economy=self.economy, block_reward=self.block_reward, block_producer=self.block_producer,
+            )
+
+            month_reward = \
+                (block_producer_reward.get() + active_block_producer_reward.get()) * self.economy.blocks_per_month
+
+            roi['statistics_per_month'].append({
+                'month': month,
```
Comment (anastasiia-bilova) [inline calculator/domain/roi.py:57]: Add a month reward.

File: calculator/domain/roi.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+                'token_price': self.economy.token_price,
+                'token_price_growth_percent': token_percent_growth,
+                'block_producer_stake': self.block_producer.stake,
+
```
Comment (anastasiia-bilova) [inline calculator/domain/roi.py]: Remove blank line.

File: calculator/forms.py [hunk 4]
```diff
@@ -18,3 +18,16 @@ class CalculateInvestmentsPaybackPerMonth(Schema):
     active_block_producers_votes = fields.Integer()
     stake = fields.Integer()
     votes = fields.Integer()
+
+
+class CalculateRoi(Schema):
+    """
+    Calculate returning of investments.
```
Comment (anastasiia-bilova) [inline calculator/forms.py]:
```suggestion
    Calculate returning on investment.
```

File: tests/test_server.py [hunk 5]
```diff
@@ -100,3 +100,354 @@ def test_calculate_investments_payback_with_invalid_arguments(client):
 
     assert expected_error == response.json
     assert HTTPStatus.BAD_REQUEST == response.status_code
+
+
+def test_calculate_roi(client):
+    """
+    Case: calculate returning on investments for 4 years.
```
Comment (anastasiia-bilova) [inline tests/test_server.py]:
```suggestion
    Case: calculate returning on investment for 4 years.
```

**Review by anastasiia-bilova — APPROVED** [review]

File: calculator/domain/roi.py (hunk 1 above)
Comment (dmytrostriletskyi) [reply calculator/domain/roi.py]: @anastasiia-bilova, done.

File: calculator/domain/roi.py (hunk 2 above)
Comment (dmytrostriletskyi) [reply calculator/domain/roi.py:57]: @anastasiia-bilova, done.

File: calculator/domain/roi.py (hunk 3 above)
Comment (dmytrostriletskyi) [reply calculator/domain/roi.py]: @anastasiia-bilova, done.

File: calculator/forms.py (hunk 4 above)
Comment (dmytrostriletskyi) [reply calculator/forms.py]: @anastasiia-bilova, done.

File: tests/test_server.py (hunk 5 above)
Comment (dmytrostriletskyi) [reply tests/test_server.py]: @anastasiia-bilova, done.

**Pull request merged**